build/common.props

```diff
@@ -0,0 +1,23 @@
+<Project>
+  <Import Project="..\version.props" />
```

Comment:
Not quite. Relative paths in imported projects are interpreted relative to the directory of the importing project.